node/runtime/src/system.rs

```diff
@@ -384,7 +388,8 @@ mod tests {
     fn test_refund_on_send_money_to_non_existent_account() {
         let (runtime, trie, root) = get_runtime_and_trie();
         let (mut alice, root) = User::new(runtime, &alice_account(), trie.clone(), root);
-        let (new_root, apply_results) = alice.send_money(root, &eve_account(), 10);
+        let money_used = 10;
```

Comment:
It seems that `money_used` is not necessary here.

Reply:
Done.